src/main/java/cuziArchitecture/driver/ProductPage.java

```diff
@@ -0,0 +1,46 @@
+package cuziArchitecture.driver;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+import java.util.List;
+
+public class ProductPage extends ParentPage {
+    WebDriver driver;
+
+    @FindBy(css = ".mb-3")
+    List<WebElement> products;
```

Comment:
Good job on the locators. Some context for you. If a locator is unique to a particular page on the web, and no other page object has access to these areas. You'll never call these locators anywhere else but the designated page.

So in this instance. Lets take checkoutEle for example.
```
@FindBy(css = ".mb-3")
List<WebElement> products;
```
You could make this locator private so that no other page object or other classes have access to them. You can do this like so.
```
@FindBy(css = ".mb-3")
private List<WebElement> products;
```
Do this for all the page objects under your construction for every locator.